rust/lance/src/index/vector/ivf/io.rs

```diff
@@ -54,7 +54,7 @@ pub(super) async fn write_index_partitions(
                     .expect("part id column not found")
                     .as_primitive();
                 let part_id = part_ids.values()[0];
-                streams_heap.push((part_id, new_streams.len()));
+                streams_heap.push((u32::MAX - part_id, new_streams.len()));
```

Comment:
is this for minheap?
Can we just use https://doc.rust-lang.org/std/collections/struct.BinaryHeap.html#min-heap?